DOCS-8201 Add Datadog Costs Documentation (#23655)
DataDog/documentation: content/en/cloud_cost_management/_index.md

    tag: "Documentation"
    text: "Learn about Tag Pipelines"
  - link: "/cloud_cost_management/tag_pipelines"
    tag: "Documentation"
    text: "Standardize tags across Cloud Cost Management with Tag Pipelines"
cascade:
    algolia:
      subcategory: 'Cloud Cost Management'
      rank: 70
      tags: ['cloud cost', 'cloud integrations', 'cloud cost management', 'cloud cost aws', 'cloud cost azure', 'cloud cost google cloud', 'cloud cost gcp', 'data collected aws', 'data collected azure', 'data collected google cloud']
---

{{< site-region region="gov" >}}
<div class="alert alert-warning">Cloud Cost Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
{{< /site-region >}}

## Overview

Cloud Cost Management provides insights for engineering and finance teams to understand how infrastructure changes impact costs, allocate spend across your organization, and identify inefficiencies.

{{< img src="cloud_cost/overview.png" alt="Gain insights into your cloud provider's cost and usage on the Cloud Costs Overview page in Datadog" style="width:100%;" >}}

> Datadog ingests your cloud cost data and transforms them into metrics that you can use in a search query on the [**Analytics** page][1]. If costs rise, you can correlate the increase with usage metrics to determine the root cause.

Reviewer: ```suggestion
Datadog ingests your cloud cost data and transforms it into metrics you can use in a search query on the [**Analytics** page][1]. If costs rise, you can correlate the increase with usage metrics to determine the root cause.
```

## Use cloud cost data

Visualize infrastructure spend alongside related utilization metrics to spot potential inefficiencies and savings opportunities.

When creating a dashboard, select **Cloud Cost** as the data source for your search query.

{{< img src="cloud_cost/cloud_cost_data_source.png" alt="Cloud Cost available as a data source in dashboard widget creation" style="width:100%;" >}}

Optionally, you can programmatically export a timeseries graph of your cloud cost data by using the [Metrics API][2].

## Create tag rules

Use [Tag Pipelines][5] to ensure comprehensive cost tracking by standardizing the tags across all cloud resources. This prevents any cost data from being overlooked. 

{{< img src="cloud_cost/tags_addnew.png" alt="Create a tag rule in Tag Pipelines to ensure your cloud resources use standard tags" style="width:60%;" >}}

You can create tag rules to correct missing or incorrect tags and add inferred tags that align with your organization's business logic.

## Create a cost monitor

Proactively manage and optimize your cloud spending by creating a [Cloud Cost Monitor][3]. You can choose **Cost Changes** or **Cost Threshold** to monitor your cloud expenses.

> {{< img src="cloud_cost/monitor.png" alt="Create a Cloud Cost monitor that alerts on changes in your cost" style="width:100%;" >}}

Reviewer: ```suggestion
{{< img src="cloud_cost/monitor.png" alt="Create a Cloud Cost monitor that alerts on cost changes" style="width:100%;" >}}
```
